### PR TITLE
webui: introduce a job timeline chart

### PR DESCRIPTION
**Pull request by fbergkemper**
### Thank you for contributing to the Bareos Project!

This PR introduces a job timeline chart.

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - webui and console: add job duration column in job listings [PR #1007]
 - packages: Build EL_8 packages instead of CentOS_8 for RHEL 8 derivates (Alma, Rocky, CentOS Stream)[PR #992]
 - webui: provide a notification that NDMP restores are NOT supported by webui [PR #1020]
+- webui: introduce a job timeline chart (preview) [PR #1017]
```
Comment (pstorz) [inline CHANGELOG.md:111]: I think we can remove the "(preview)" both from the PR and the description.

**Review by pstorz — CHANGES_REQUESTED** [review]
So far looks good. I am missing a legend that describes what the colors in the timeline mean.

File: CHANGELOG.md (hunk 1 above)
Comment (fbergkemper) [reply CHANGELOG.md]: Removed it.

**Comment by fbergkemper** [comment]
> So far looks good. I am missing a legend that describes what the colors in the timeline mean.

I've added a legend underneath the chart.

**Review by pstorz — APPROVED** [review]
Good work, I commited the changes by bareos-check-sources.

**Pull request merged**